tests/Common/BaseTestCase.php

```diff
@@ -36,9 +36,22 @@ protected function ddTearDown()
         if (\class_exists('DDTrace\Log\Logger')) {
             Logger::reset();
         }
+        foreach ($this->envsToCleanUpAtTearDown() as $env) {
+            self::putEnv($env . '=');
```

Comment:
```suggestion
            self::putEnv($env);
```
`$env=` sets it to empty, `$env` only sets it to undefined, i.e. default